flag.txt

```diff
@@ -1 +1 @@
-Sat May 27 15:14:07 UTC 2023
+Sat May 27 15:17:29 UTC 2023
```

Comment:
We don't need the flag.txt anymore, since we use the keep-alive action.

Reply:
Thanks @andrew2net , removed flag.txt